articles/ai-safety/index.md

```diff
@@ -0,0 +1,72 @@
+
+Even if you make a good goal for it, it can still find a way to cheat the system. [There are many examples of this happening](https://www.youtube.com/watch?v=nKJlF-olKmg), often hilariously. Say you have an AI that you want to clean a room. It gets rewarded whenever it looks around the room and doesn't see a mess. In response, it puts a bucket on its head, so it doesn't see a mess anymore.
+
+## What Do We Do?
+
+### Possible Solutions?
+
+[There](http://intelligence.org/summary) [are](http://cser.org/emerging-risks-from-technology/artificial-intelligence/) [entire](http://www.fhi.ox.ac.uk/) [organizations](https://www.partnershiponai.org/) [dedicated](http://gcrinstitute.org/) [to](http://futureoflife.org/static/data/documents/FLI-XRisk-Organizations.pdf?x59035) [this](https://80000hours.org/career-guide/top-careers/profiles/artificial-intelligence-risk-research/). To try to give the definitive solution would be very vain. But, there may be some good leads in the following areas.
+
+* **Involving humans in the process.** You could create an AI that only tells a human of the decisions it makes. It would have to have a pretty realistic simulation of the world to work in. It should never simulate the action of telling a human to make a certain decision. You could also make an AI where all its actions must first be approved, but that would be terribly slow.
+* **Give the AI an incentive to value ethics.** This would be a great idea if we could all agree on what ethics involves. It doesn't work. [There are many moral philosophies](https://en.wikipedia.org/wiki/Ethics). None of them are perfect.
+* **Make an AI and tell it to maximize [QALYs](https://en.wikipedia.org/wiki/Quality-adjusted_life_year).** Utilitarianism is pretty popular, but it still has [problems](https://www.thinkingmuchbetter.com/main/5-objections-to-utilitarianism). It also still has the problem of [population ethics](https://www.youtube.com/watch?v=vqBl50TREHU). Also, as mentioned before, AIs can take advantage of their reward function. If this AI was implemented, all it would do is maximize the number of babies born.
+
+### Organizations to Donate To
+
+[In](https://80000hours.org/) [Effective](https://www.evidenceaction.org/) [Altruism](https://www.gfi.org/), [we](https://d-rev.org/) [like](https://www.givedirectly.org/) [to](https://www.gofundme.com/f/covidmoonshot) [focus](https://www.ciwf.com/) [on](https://aisafetycamp.com/) [organizations](https://www.againstmalaria.com/) [to](https://allfed.info/) [donate](https://www.charitywater.org/) [to](https://www.arborday.org/). Here's a shortlist of places to look at:
```

Comment:
Its not clear why you have inserted links in the first sentence. Each word has a different link. Do you want to emphasize on that? I don't quite understand that. If they are links to organizations, then probably add them in the list below. If the list is too big, filter it out accordingly.